pgxn/neon_walredo/walredoproc.c

```diff
@@ -140,9 +140,17 @@ static XLogReaderState *reader_state;
 #define TRACE DEBUG5
 
 #ifdef HAVE_LIBSECCOMP
+
+#include <linux/close_range.h>
+
 static void
 enter_seccomp_mode(void)
 {
+
+	if (int err = close_range(3, ~0U, 0)) {
```

Comment:
Let's add a comment to explain why we're doing this, and a corresponding comment on the pageserver side to explain that we're safe launching without .close_fds() because this code exists.

Reply:
Sigh, had opened the PR with an outdated commit.

Latest force push contains proper commentary.